l10n_it_central_journal/.gitignore

```diff
@@ -0,0 +1,59 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
what's the reason for this?

Reply:
@tafaRU right, removeing: https://github.com/OCA/l10n-italy/pull/430